.eslintrc.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
     env: {
         browser: true,
-        es6: true
+        es6: true,
+        jest: true,
```

Comment:
Note: This is to stop eslint from complaining at `console-utils.test.js` that "describe" is not defined